lib/Config.php

```diff
@@ -162,7 +183,7 @@ public function get_connection($name)
 	 *
 	 * @return string
 	 */
-	public function get_default_connection_string()
+	public function get_default_connection_info()
```

Comment:
This is a public API change. While I agree the new name makes more sense, this will be a small BC break. Should definitely be noted somewhere.